CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 #### Changes
 
 - `andrew` is updated to `0.3`.
+- `create_surface` and `create_surface_with_scale_callback` now return `Attached<WlSurface>`
```

Comment:
This is a breaking change, so it should be marked in a `#### Breaking Changes` section.